### DISCUSSION TITLE
Will there be a part 2 of this course?

### DISCUSSION THREAD

**Discussion by longtongster** [mrdbourke/pytorch-deep-learning#345] [General]
This is one of the best courses I have done. On udemy it says that there will be a part 2. Can you confirm that my understanding is correct? That would really be great ...

**Comment by mrdbourke** [comment]
Hi @longtongster,

Thank you for the kind words!

Can you let me know where it says there will be a part 2 on Udemy? 

I'm unaware of this.

If enough people would like a part 2, then it is something I'd consider. But as of now, there are no plans for a *full* part 2.

Extensions are more likely.

Is there anything you'd like to see?

**Reply by longtongster** [reply]
Hi @mrdbourke,

In section 13 the bonus lecture it states 

"If you enjoyed this course you may enjoy Part 2 of this course!"

So that is why I posted the question. For me there are still some fields that interest me that would fit in a Part 2 or extensions:

1. How to create/work with datasets that are too big to fit into memory
2. Working with callbacks
3. Do some distributed training with pytorch (train a imagenet lite or something like that)
4. Build a transformer model from scratch in pytorch
5. Build and use a vision transformer
6. Build under your guidance a Yolo like object detector and or other state of the art more complex architectures. 

I hope these make sense. 

I would appreciate any info if you do something in that direction. I definitely take a part 2 of the course ... learning the pytroch song part 2 :-)

**Reply by longtongster** [reply]
well I see you have removed the announcement for part 2 on the course from udemy. Then I guess you have to extend the part 1 of the course :-)
